<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FOImportGui</class>
 <widget class="QWidget" name="FOImportGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="8" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mesh File Location</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="sex">
          <property name="text">
           <string>Patient Sex</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="heelraise">
          <property name="text">
           <string>Heel Raise</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="Gui::FileChooser" name="browse" native="true">
          <property name="toolTip">
           <string>Import File Location</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="text" stdset="0">
           <string>Browse Files</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="sexOut">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>Female</string>
          </property>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="posting">
          <property name="text">
           <string>Posting (+ medial, - lateral)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="thickness">
          <property name="text">
           <string>Thickness of FO</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="thicknessOut">
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="size">
          <property name="text">
           <string>Shoe Size</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="postingOut">
          <property name="suffix">
           <string> degrees</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>-10</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="sizeOut">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8.5</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="heelraiseOut">
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::FileChooser</class>
   <extends>QWidget</extends>
   <header>Gui/FileDialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
